vtexdocs/devportal
# chore: update rapidoc to show postman button in overview

#### What is the purpose of this pull request?

To update the RapiDoc submodule.

#### What problem is this solving?

The postman download button did not appear in overview pages.

#### How should this be manually tested?

Open any API Reference overview pages in the preview and check if the buttons appear on the top of the page.

#### Screenshots or example usage

<img width="676" alt="image" src="https://github.com/vtexdocs/devportal/assets/62757720/6b733f49-b585-42ae-9ba2-a716e8aeb5c9">

#### Types of changes
